src/DetailView/DetailView.js

```diff
@@ -1,53 +1,73 @@
+import { keyboard } from "@testing-library/user-event/dist/keyboard"
```

Comment:
Curious about this, since I don't see a `keyboard` variable being used anywhere. Is this for a future iteration or testing?

Reply:
I'm actually not sure what that is either, I actually thought that was something you had added during accessibility but I never went back and checked. Looks like we can get rid of it, I'm not really sure how it ended up there. Good catch.

Reply:
Removed in latest PR.